spec/controllers/account/projects_controller_spec.rb

```diff
@@ -69,6 +69,13 @@
       end
     end
 
+    describe 'GET /projects/search' do
+      it 'should return search result in format json' do

```

Comment:
test is not complete

your DB is blank (are you sure it returns something?)

Reply:
at least not empty response

Reply:
json requests should be validated
better to move them into a requests tests

Reply:
Roman what is "request test"? 

Maybe let's not overcomplicate things :) as suggested add validation to check that keys in response are the same as expected.